frontend/public/models/yaml-templates.ts

```diff
@@ -693,4 +693,48 @@ spec:
     type: JenkinsPipeline
   triggers:
   - type: ConfigChange
+`).setIn([referenceForModel(k8sModels.ResourceQuotaModel), 'rq-compute'], `
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: compute-quota
+  namespace: test-ns
+spec:
+  hard:
+    pods: '4'
```

Comment:
I'd leave off the object count here since the example is focused on compute resources